client/src/components/common/boost/BoostCalibration.stories.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { addArgs, createStory } from 'utils/stories';
+import BoostCalibration from './BoostCalibration';
+
+export default {
+    title: 'BoostCalibration',
+    component: BoostCalibration,
+  };
+
+  const Template = addArgs<any>((props) => (
+    <BoostCalibration {...props} />
```

Comment:
Once you've got the buttons calling callbacks passed in as props, supply storybook `action`s as the callbacks so that we can see when the callbacks are triggered from storybook. (See `BoostConfigList.stories.tsx` for an example)